repo_guide.md

```diff
@@ -0,0 +1,45 @@
+
+We have a webserver running, which currently has several APIs implemented (see [server](server)), including
+
+1. `/packages`: Using [https://c107-224.cloud.gwdg.de/packages](https://c107-224.cloud.gwdg.de/packages), one receives a JSON-list of packages available at the server. Each JSON entry has a title, a description, a last-modified date and a package version.
+2. Under `<package_name>/zip_file`, one can download a package as a zip-file, for example [https://c107-224.cloud.gwdg.de/zip_file/2020_Yu_NorthRussia](https://c107-224.cloud.gwdg.de/zip_file/2020_Yu_NorthRussia).
+
+The webserver is powered by an HTTP server program called `poseidon-http-server` (see also [server](server)), whose source-code can be reviewed at the [same repository](https://github.com/poseidon-framework/poseidon-hs) as `trident` is sourced. For those interested to run their own instance of this webserver, for example under `localhost`, the server comes with a short command line help. It gets installed via `stack install`, similarly to `trident`. The program first scans the given directories for Poseidon packages, then creates zip-files for them, and then starts a HTTP server listening - by default - to port 3000, and providing the two APIs listed above.
+
+## Contributing to our central repositories
+
+The Poseidon framework has a strongly decentralized philosophy and relies very much on a community of users willing to prepare and improve the data in the repositories. To ensure a professional and welcoming atmosphere we would first of all like to ask you to respect our [Contributor Code of Conduct](conduct.md) in all interactions with the Poseidon team and other users on Github and beyond.
+
+There are two basic ways you can contribute to Poseidon:
+
+1) If you identify a mistake in any package metadata, be it context data (`.janno`-files), package-meta-data (`POSEIDON.yml`) or bibliographic information (`.bib` files), we welcome contributions to correct or extend that data. This goes most easily through our github-package, which you can fork, commit changes and ask for Pull requests. You can also use the [issue tracker](https://github.com/poseidon-framework/published_data/issues) on github (for which you can find some help [here](https://lab.github.com/githubtraining/introduction-to-github)). See below for more details on this.
+
+2) If you would like to provide a new package, we would welcome you preparing the meta-data for that new package as far as you can and commiting this package - without the genotype data - as a pull request to our [github-repo](https://github.com/poseidon-framework/published_data). The genetic data we would like to process ourselves, to ensure maximum compatibility with the existing database and to minimise batch effects. So if you want us to add an entirely new package with genetic data, please share with us the raw data, such as BAM or fastq-files, so that we can generate the data and include it into a new Poseidon package.
```

Comment:
This is not entirely true **at the moment**, we are getting a lot of data from another lab's processing. We don't have the exact pipeline with Boston lab, so what is the difference? Wouldn't we allow genotype data from another group if they share their pipeline with us? 
Unless this is our long term plan, to process everything ourselves (which is a better option in my opinion).

Reply:
Well, I think before going public we need to make a clear decision here, whether we accept foreign genotype data or not. My understanding was that 
i) we currently have only in-house processed data, or data from Boston in our public repo. Is this correct?
ii) that we agreed that we would like to keep it that way, so not accept foreign generated genotype data in order to not introduce dangerous heterogeneity and batch effects (Technically, that point can already be made with our hybrid Boston/Jena dataset, but there is some level of pragmatism that we can't escape. We simply can't rerun everything through Jena)

So, I think what I wrote reflects what we decided last meeting, that we would like to process genetic data ourselves from BAM files.

Reply:
i)Yes, that is correct.
I agree, my comment regarding the second part was more of a longer term vision for us.